test/integration/test_create_dss_docker_image.py

```diff
@@ -38,7 +47,23 @@ def dss_docker_container():
     finally:
         container.stop()
         container.remove()
-        client.images.remove(testee.image_name)
+
+
+def request_with_retry(url: str, timeout: timedelta, verbose:bool = False) -> requests.Response:
```

Comment:
https://tenacity.readthedocs.io/en/latest/ already implements that, and we use it already in other projects